cc/main.cc

```diff
@@ -65,6 +66,8 @@ DEFINE_uint64(seed, 0,
               "Random seed. Use default value of 0 to use a time-based seed. "
               "This seed is used to control the moves played, not whether a "
               "game has resignation disabled or is a holdout.");
+DEFINE_bool(even, false,
```

Comment:
this should probably be the default.  i don't think there's any reason not to do this, if parallel_games > 1

Reply:
Done. For parallel_games == 1, it will just work as before.